opencti-platform/opencti-front/src/components/ExportButtons.jsx

```diff
@@ -179,28 +178,21 @@ class ExportButtons extends Component {
           {handleDashboardDuplication && variant === 'dashboard' && (
             <Tooltip title={t('Duplicate the dashboard')}>
               <ToggleButton
-                sx={{ padding: '2px' }}
                 size="small"
                 value="duplicate-dashboard"
+                onClick={handleDashboardDuplication.bind(this)}
               >
-                <IconButton
-                  aria-label="copy"
-                  onClick={handleDashboardDuplication.bind(this)}
-                  color="primary"
-                  size="small"
-                >
-                  <ContentCopyOutlined fontSize="small" />
-                </IconButton>
+                <ContentCopyOutlined fontSize="small" color="primary" />
               </ToggleButton>
             </Tooltip>
           )}
           <Tooltip title={t('Export to image')}>
-            <ToggleButton onClick={this.handleOpenImage.bind(this)}>
+            <ToggleButton onClick={this.handleOpenImage.bind(this)} value={'Export-to-image'} >
```

Comment:
just curious: why is the value this string of characters, why do we add the "-" between the words?